client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -667,30 +720,279 @@ class WpcomChecklistComponent extends PureComponent {
 			/>
 		);
 	};
+
+	renderAboutTextUpdatedTask = ( TaskComponent, baseProps, task ) => {
+		const { translate, viewMode, taskStep } = this.props;
+		const isPrompt = viewMode === 'prompt';
+		const taskProps = Object.assign( {}, baseProps, {
+			title: translate( 'Edit About text' ),
+			description: translate(
+				'Update the text we’ve written for you to describe what makes your business unique. ' +
+					'Make your homepage speak to your customers.'
+			),
+			duration: translate( '%d minute', '%d minutes', { count: 5, args: [ 5 ] } ),
+			onClick: this.handleInlineHelpProgress( task, isPrompt ),
+			onDismiss: this.handleTaskDismiss( task.id ),
+			isPrompt,
+		} );
+
+		if ( isPrompt ) {
+			taskProps.onDismiss = this.handleInlineHelpDismiss( task.id, { maxStep: 2 } );
+			taskProps.canDismiss = taskStep !== 1;
+			taskProps.autoCloseOnAction = false;
+
+			switch ( taskStep ) {
+				case 0:
+					taskProps.buttonText = translate( 'Try it' );
+					taskProps.dismissButtonText = translate( 'Mark complete' );
+					break;
+				case 1:
+					taskProps.title = translate( 'Update your homepage text' );
+					taskProps.description = translate(
+						'Task a moment to review what we’ve written for you. ' +
+							'Click the text to make any additions or changes.'
+					);
+					taskProps.onClick = taskProps.onDismiss;
+					taskProps.buttonText = translate( 'Done editing' );
+					taskProps.duration = null;
+					break;
+				case 2:
+					taskProps.title = translate( 'Homepage updated!' );
+					taskProps.buttonText = translate( 'Next task' );
+					taskProps.dismissButtonText = translate( 'View all tasks' );
+					taskProps.onClick = this.handleInlineHelpNext;
+					taskProps.onDismiss = this.backToChecklist;
+					taskProps.duration = null;
+					taskProps.autoCloseOnAction = true;
+					break;
+			}
+		}
+
+		return <TaskComponent { ...taskProps } />;
+	};
+
+	renderHomepagePhotoUpdatedTask = ( TaskComponent, baseProps, task ) => {
+		const { translate, viewMode, taskStep } = this.props;
+		const isPrompt = viewMode === 'prompt';
+		const taskProps = Object.assign( {}, baseProps, {
+			title: translate( 'Change homepage photo' ),
+			description: translate(
+				'Upload your own photoo or choose from a wide selection of free ones to personalize your new site.'
```

Comment:
Photo~o~

Reply:
Now feeling a bit ashamed of the typos. 😅